src/azure-cli/azure/cli/command_modules/container/tests/latest/test_container_commands.py

```diff
@@ -227,7 +227,7 @@ def test_container_create_with_acr(self, resource_group, resource_group_location
         registry_username = 'clitestregistry1'
         registry_server = '{}.azurecr.io'.format(registry_username)
         image = '{}/nginx:latest'.format(registry_server)
-        password = '5+36OCtbIwfy8g5glC4bQQrFsfmMc3iD'
+        password = 'passplaceholder'
```

Comment:
How dose the test get the real password after replacing it by placeholder?

Reply:
@joseph-porter, this is a `record_only` test, did you run it lively?

Reply:
@jsntcy  I did run the test live. It is record_only because it requires an additional resource to be created, which in this case is an Azure Container Registry with an uploaded image.

@zhoxing-ms Each test we have to regenerate the password in the ACR, so we regenerate the password on the ACR resource and insert it here for a single test run.